graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/ddl/fetchers/KeyspaceDto.java

```diff
@@ -88,7 +89,8 @@ private boolean isAuthorized(Table table) {
           authenticationSubject,
           Collections.singletonList(keyspace.name()),
           Collections.singletonList(table.name()),
-          SourceAPI.GRAPHQL);
+          SourceAPI.GRAPHQL,
+          ResourceKind.KEYSPACE);
```

Comment:
`ResourceKind.TABLE`?

Reply:
Fixed. I got overzealous with the copy paste :)